templates/csharp-netcore/ApiClient.mustache

```diff
@@ -0,0 +1,724 @@
+{{>partial_header}}
```

Comment:
Which parts of these templates did you have to edit? Just for my own curiosity

Reply:
One was to update how enums were converted to strings and the other was to update how the body of the requests was rendered